views/auth/signup.html

```diff
@@ -50,9 +41,7 @@ <h1 style="{{if .FlashSuccess}}display: none;{{end}}">Sign Up</h1>
   </div>
 
   <div class="form-group col-sm-7 pull-right" style="{{if .FlashSuccess}}display: none;{{end}}">
-<!--    <label class="col-md-4 control-label" for="reset"></label> -->
         <p><a href="/passwordreset">Forgot your password?</a></p>
-        <p><a href="/signup">New user?</a></p>
```

Comment:
Perhaps this should be `<p><a href="/signin">Already registered?</a></p>`, but I don't think it's really necessary.